test/fixture/accept_very_simple.email

```diff
@@ -0,0 +1,7 @@
+From: accept_imple@example.com
+To: junkword.test@example.net
+Subject: this is a mail which should be accepted
+
+Simple acceptable mail. We assume that the "test" account is created,
+has no watchwords (or matches word) and hasn't had mails come into
+junkword.
```

Comment:
not particularly, though I guess it doesn't matter as such. I think we should have trailing new line in all cases on standard files. Otherwise things are a big ugly.